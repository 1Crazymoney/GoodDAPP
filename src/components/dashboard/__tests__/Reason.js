import React from 'react'
import { getWebRouterComponentWithMocks } from './__util__'
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

describe('Reason', () => {
  it('renders without errors', () => {
    const Reason = getWebRouterComponentWithMocks('../Reason')
    const tree = renderer.create(<Reason />)
    expect(tree.toJSON()).toBeTruthy()
  })

  it('matches snapshot', () => {
    const Reason = getWebRouterComponentWithMocks('../Reason')
    const component = renderer.create(<Reason />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
