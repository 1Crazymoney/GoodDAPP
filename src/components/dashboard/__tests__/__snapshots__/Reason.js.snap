// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reason matches snapshot 1`] = `
<div
  className="css-view-1dbjc4n"
  style={
    Object {
      "WebkitBoxFlex": 1,
      "WebkitFlexBasis": "0%",
      "WebkitFlexGrow": 1,
      "WebkitFlexShrink": 1,
      "flexBasis": "0%",
      "flexGrow": 1,
      "flexShrink": 1,
      "msFlexNegative": 1,
      "msFlexPositive": 1,
      "msFlexPreferredSize": "0%",
    }
  }
>
  <div
    className="css-view-1dbjc4n"
    style={
      Object {
        "WebkitBoxDirection": "normal",
        "WebkitBoxFlex": 1,
        "WebkitBoxOrient": "vertical",
        "WebkitBoxPack": "start",
        "WebkitFlexBasis": "0%",
        "WebkitFlexDirection": "column",
        "WebkitFlexGrow": 1,
        "WebkitFlexShrink": 1,
        "WebkitJustifyContent": "flex-start",
        "backgroundImage": "linear-gradient(to bottom, #00AFFF, #2DC0F7, #28C0EF, #23C0E7, #1EC1DF, #19C1D7, #14C1CF, #0FC2C7, #0FC2C7, #0AC2BF, #05C2B7, #00C3AF)",
        "display": "flex",
        "flexBasis": "0%",
        "flexDirection": "column",
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "flex-start",
        "msFlexDirection": "column",
        "msFlexNegative": 1,
        "msFlexPack": "start",
        "msFlexPositive": 1,
        "msFlexPreferredSize": "0%",
        "paddingBottom": "8px",
        "paddingLeft": "8px",
        "paddingRight": "8px",
        "paddingTop": "8px",
        "width": "100%",
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "backgroundColor": "rgba(255,255,255,1.00)",
          "borderBottomLeftRadius": "5px",
          "borderBottomRightRadius": "5px",
          "borderTopLeftRadius": "5px",
          "borderTopRightRadius": "5px",
          "marginBottom": "8px",
          "paddingBottom": "8px",
          "paddingLeft": "8px",
          "paddingRight": "8px",
          "paddingTop": "8px",
        }
      }
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitAlignItems": "center",
            "WebkitBoxAlign": "center",
            "WebkitBoxDirection": "normal",
            "WebkitBoxOrient": "horizontal",
            "WebkitBoxPack": "justify",
            "WebkitFlexDirection": "row",
            "WebkitJustifyContent": "space-between",
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "space-between",
            "msFlexAlign": "center",
            "msFlexDirection": "row",
            "msFlexPack": "justify",
          }
        }
      >
        <div
          aria-disabled={true}
          className="css-view-1dbjc4n"
          disabled={true}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "MozTransitionProperty": "opacity",
              "MozUserSelect": "none",
              "WebkitTransitionDuration": "0.15s",
              "WebkitTransitionProperty": "opacity",
              "WebkitUserSelect": "none",
              "backgroundColor": "rgba(0,0,0,0.00)",
              "msUserSelect": "none",
              "transitionDuration": "0.15s",
              "transitionProperty": "opacity",
              "userSelect": "none",
            }
          }
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "backgroundColor": "rgba(255,255,255,1.00)",
                "borderBottomColor": "rgba(112,112,112,1.00)",
                "borderBottomLeftRadius": "17px",
                "borderBottomRightRadius": "17px",
                "borderBottomWidth": "1px",
                "borderLeftColor": "rgba(112,112,112,1.00)",
                "borderLeftWidth": "1px",
                "borderRightColor": "rgba(112,112,112,1.00)",
                "borderRightWidth": "1px",
                "borderTopColor": "rgba(112,112,112,1.00)",
                "borderTopLeftRadius": "17px",
                "borderTopRightRadius": "17px",
                "borderTopWidth": "1px",
                "height": "34px",
                "width": "34px",
              }
            }
          >
            <div
              className="css-view-1dbjc4n"
              style={
                Object {
                  "WebkitFlexBasis": "auto",
                  "borderBottomLeftRadius": "17px",
                  "borderBottomRightRadius": "17px",
                  "borderTopLeftRadius": "17px",
                  "borderTopRightRadius": "17px",
                  "flexBasis": "auto",
                  "height": "34px",
                  "msFlexPreferredSize": "auto",
                  "overflowX": "hidden",
                  "overflowY": "hidden",
                  "width": "34px",
                  "zIndex": 0,
                }
              }
            >
              <div
                className="css-view-1dbjc4n"
                style={
                  Object {
                    "backgroundColor": "rgba(0,0,0,0.00)",
                    "backgroundPosition": "center",
                    "backgroundRepeat": "no-repeat",
                    "backgroundSize": "cover",
                    "bottom": "0px",
                    "height": "100%",
                    "left": "0px",
                    "position": "absolute",
                    "right": "0px",
                    "top": "0px",
                    "width": "100%",
                    "zIndex": -1,
                  }
                }
              />
            </div>
          </div>
        </div>
        <div
          className="css-view-1dbjc4n"
          style={
            Object {
              "display": "inline-block",
            }
          }
        >
          <div
            className="css-text-901oao"
            dir="auto"
            style={
              Object {
                "color": "rgba(85,85,85,1.00)",
                "direction": "ltr",
                "fontFamily": "Roboto",
                "fontSize": "30px",
                "fontWeight": "normal",
                "lineHeight": "22px",
                "textAlign": "right",
                "textDecoration": "none",
                "textTransform": "none",
              }
            }
          >
            -.--
          </div>
          <div
            className="css-text-901oao"
            dir="auto"
            style={
              Object {
                "color": "rgba(85,85,85,1.00)",
                "direction": "ltr",
                "fontFamily": "Roboto",
                "fontSize": "18px",
                "fontWeight": "normal",
                "lineHeight": "22px",
                "textAlign": "right",
                "textDecoration": "none",
                "textTransform": "none",
              }
            }
          >
            G$
          </div>
        </div>
      </div>
    </div>
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitBoxFlex": 1,
          "WebkitFlexBasis": "0%",
          "WebkitFlexGrow": 1,
          "WebkitFlexShrink": 1,
          "backgroundColor": "rgba(255,255,255,1.00)",
          "borderBottomLeftRadius": "5px",
          "borderBottomRightRadius": "5px",
          "borderTopLeftRadius": "5px",
          "borderTopRightRadius": "5px",
          "flexBasis": "0%",
          "flexGrow": 1,
          "flexShrink": 1,
          "msFlexNegative": 1,
          "msFlexPositive": 1,
          "msFlexPreferredSize": "0%",
          "paddingBottom": "8px",
          "paddingLeft": "8px",
          "paddingRight": "8px",
          "paddingTop": "8px",
        }
      }
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitAlignItems": "center",
            "WebkitBoxAlign": "center",
            "WebkitBoxDirection": "normal",
            "WebkitBoxOrient": "vertical",
            "WebkitBoxPack": "justify",
            "WebkitFlexDirection": "column",
            "WebkitJustifyContent": "space-between",
            "alignItems": "center",
            "flexDirection": "column",
            "height": "100%",
            "justifyContent": "space-between",
            "msFlexAlign": "center",
            "msFlexDirection": "column",
            "msFlexPack": "justify",
          }
        }
      >
        <div
          className="css-view-1dbjc4n"
          style={
            Object {
              "WebkitBoxDirection": "normal",
              "WebkitBoxOrient": "vertical",
              "WebkitBoxPack": "justify",
              "WebkitFlexDirection": "column",
              "WebkitJustifyContent": "space-between",
              "flexDirection": "column",
              "justifyContent": "space-between",
              "msFlexDirection": "column",
              "msFlexPack": "justify",
              "width": "100%",
            }
          }
        >
          <div
            className="css-text-901oao"
            dir="auto"
            style={
              Object {
                "color": "rgba(85,85,85,1.00)",
                "direction": "ltr",
                "fontFamily": "Roboto",
                "fontSize": "24px",
                "fontWeight": "500",
                "lineHeight": "22px",
                "marginBottom": "1rem",
                "textAlign": "center",
                "textDecoration": "none",
                "textTransform": "uppercase",
              }
            }
          >
            What For?
          </div>
          <input
            autoCapitalize="sentences"
            autoComplete="on"
            autoCorrect="on"
            autoFocus={true}
            className="css-textinput-1cwyjr8"
            data-focusable={true}
            dir="auto"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyPress={[Function]}
            onSelect={[Function]}
            placeholder="Add a message"
            readOnly={false}
            spellCheck={true}
            style={
              Object {
                "borderBottomColor": "rgba(66,69,74,1.00)",
                "borderBottomStyle": "solid",
                "borderBottomWidth": "1px",
                "color": "rgba(66,69,74,1.00)",
                "fontSize": "18px",
                "fontStyle": "RobotoSlab-Regular",
                "paddingBottom": "8px",
                "paddingLeft": "8px",
                "paddingRight": "8px",
                "paddingTop": "8px",
                "textAlign": "center",
              }
            }
            styles={
              Object {
                "input": Object {
                  "borderBottomColor": "#42454A",
                  "borderBottomStyle": "solid",
                  "borderBottomWidth": 1,
                  "color": "#42454A",
                  "fontSize": 18,
                  "fontStyle": "RobotoSlab-Regular",
                  "padding": 8,
                  "textAlign": "center",
                },
              }
            }
            theme={
              Object {
                "colors": Object {
                  "accent": "#03dac4",
                  "backdrop": "rgba(0, 0, 0, 0.5)",
                  "background": "#f6f6f6",
                  "blue": "#006EA0",
                  "darkBlue": "#0C263D",
                  "darkGray": "#42454A",
                  "disabled": "rgba(0, 0, 0, 0.26)",
                  "error": "#B00020",
                  "gray50Percent": "#CBCBCB",
                  "green": "#00C3AE",
                  "lightGray": "#EEE",
                  "notification": "#f50057",
                  "orange": "#F8AF40",
                  "placeholder": "#CBCBCB",
                  "primary": "#00AFFF",
                  "purple": "#9F6A9D",
                  "red": "#FA6C77",
                  "surface": "#ffffff",
                  "text": "#222",
                },
                "dark": false,
                "fontStyle": Object {
                  "color": "#555",
                  "fontSize": 18,
                  "textAlign": "center",
                },
                "fonts": Object {
                  "light": "Roboto-Light, Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                  "medium": "Roboto-Medium, Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                  "regular": "Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                  "slab": "RobotoSlab-Regular",
                  "slabBold": "RobotoSlab-Bold",
                  "thin": "Roboto-Thin, Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                },
                "paddings": Object {
                  "mainContainerPadding": 8,
                },
                "roundness": 4,
                "sizes": Object {
                  "default": 8,
                  "defaultDouble": 16,
                  "defaultHalf": 4,
                },
              }
            }
            type="text"
          />
        </div>
        <div
          className="css-view-1dbjc4n"
          style={
            Object {
              "WebkitBoxDirection": "normal",
              "WebkitBoxOrient": "horizontal",
              "WebkitFlexDirection": "row",
              "flexDirection": "row",
              "marginTop": "1rem",
              "msFlexDirection": "row",
              "width": "100%",
            }
          }
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "WebkitBoxFlex": 1,
                "WebkitBoxPack": "center",
                "WebkitFlexBasis": "0%",
                "WebkitFlexGrow": 1,
                "WebkitFlexShrink": 1,
                "WebkitJustifyContent": "center",
                "backgroundColor": "rgba(0,0,0,0.00)",
                "borderBottomColor": "rgba(0,0,0,0.00)",
                "borderBottomLeftRadius": "50px",
                "borderBottomRightRadius": "50px",
                "borderBottomStyle": "solid",
                "borderBottomWidth": "0px",
                "borderLeftColor": "rgba(0,0,0,0.00)",
                "borderLeftStyle": "solid",
                "borderLeftWidth": "0px",
                "borderRightColor": "rgba(0,0,0,0.00)",
                "borderRightStyle": "solid",
                "borderRightWidth": "0px",
                "borderTopColor": "rgba(0,0,0,0.00)",
                "borderTopLeftRadius": "50px",
                "borderTopRightRadius": "50px",
                "borderTopStyle": "solid",
                "borderTopWidth": "0px",
                "boxShadow": "0px 0px 0px rgba(0,0,0,0.24)",
                "flexBasis": "0%",
                "flexGrow": 1,
                "flexShrink": 1,
                "justifyContent": "center",
                "minWidth": "auto",
                "msFlexNegative": 1,
                "msFlexPack": "center",
                "msFlexPositive": 1,
                "msFlexPreferredSize": "0%",
              }
            }
          >
            <div
              aria-disabled={true}
              className="css-view-1dbjc4n"
              disabled={true}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              role="button"
              style={
                Object {
                  "borderBottomLeftRadius": "50px",
                  "borderBottomRightRadius": "50px",
                  "borderTopLeftRadius": "50px",
                  "borderTopRightRadius": "50px",
                  "overflowX": "hidden",
                  "overflowY": "hidden",
                  "position": "relative",
                }
              }
            >
              <div
                className="css-view-1dbjc4n"
                style={
                  Object {
                    "WebkitAlignItems": "center",
                    "WebkitBoxAlign": "center",
                    "WebkitBoxDirection": "normal",
                    "WebkitBoxOrient": "horizontal",
                    "WebkitBoxPack": "center",
                    "WebkitFlexDirection": "row",
                    "WebkitJustifyContent": "center",
                    "alignItems": "center",
                    "flexDirection": "row",
                    "justifyContent": "center",
                    "msFlexAlign": "center",
                    "msFlexDirection": "row",
                    "msFlexPack": "center",
                  }
                }
              >
                <div
                  className="css-text-901oao css-textOneLine-bfa6kz"
                  dir="auto"
                  style={
                    Object {
                      "color": "rgba(163,163,163,1.00)",
                      "direction": "ltr",
                      "fontFamily": "Roboto-Medium, Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                      "letterSpacing": "1px",
                      "marginBottom": "9px",
                      "marginLeft": "8px",
                      "marginRight": "8px",
                      "marginTop": "9px",
                      "textAlign": "center",
                    }
                  }
                >
                  <div
                    className="css-view-1dbjc4n"
                    style={
                      Object {
                        "WebkitBoxPack": "center",
                        "WebkitJustifyContent": "center",
                        "display": "inline-flex",
                        "justifyContent": "center",
                        "minHeight": "34px",
                        "msFlexPack": "center",
                      }
                    }
                  >
                    <span
                      className="css-text-901oao css-textHasAncestor-16my406"
                      dir="auto"
                      style={
                        Object {
                          "direction": "ltr",
                          "fontFamily": "Roboto",
                          "fontSize": "16px",
                          "fontWeight": "normal",
                          "lineHeight": "22px",
                          "textAlign": "center",
                          "textDecoration": "none",
                          "textTransform": "none",
                        }
                      }
                    >
                      Cancel
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "WebkitBoxFlex": 2,
                "WebkitBoxPack": "center",
                "WebkitFlexBasis": "0%",
                "WebkitFlexGrow": 2,
                "WebkitFlexShrink": 1,
                "WebkitJustifyContent": "center",
                "backgroundColor": "rgba(0,0,0,0.12)",
                "borderBottomColor": "rgba(0,0,0,0.00)",
                "borderBottomLeftRadius": "50px",
                "borderBottomRightRadius": "50px",
                "borderBottomStyle": "solid",
                "borderBottomWidth": "0px",
                "borderLeftColor": "rgba(0,0,0,0.00)",
                "borderLeftStyle": "solid",
                "borderLeftWidth": "0px",
                "borderRightColor": "rgba(0,0,0,0.00)",
                "borderRightStyle": "solid",
                "borderRightWidth": "0px",
                "borderTopColor": "rgba(0,0,0,0.00)",
                "borderTopLeftRadius": "50px",
                "borderTopRightRadius": "50px",
                "borderTopStyle": "solid",
                "borderTopWidth": "0px",
                "flexBasis": "0%",
                "flexGrow": 2,
                "flexShrink": 1,
                "justifyContent": "center",
                "minWidth": "auto",
                "msFlexNegative": 1,
                "msFlexPack": "center",
                "msFlexPositive": 2,
                "msFlexPreferredSize": "0%",
              }
            }
          >
            <div
              aria-disabled={true}
              className="css-view-1dbjc4n"
              disabled={true}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              role="button"
              style={
                Object {
                  "borderBottomLeftRadius": "50px",
                  "borderBottomRightRadius": "50px",
                  "borderTopLeftRadius": "50px",
                  "borderTopRightRadius": "50px",
                  "overflowX": "hidden",
                  "overflowY": "hidden",
                  "position": "relative",
                }
              }
            >
              <div
                className="css-view-1dbjc4n"
                style={
                  Object {
                    "WebkitAlignItems": "center",
                    "WebkitBoxAlign": "center",
                    "WebkitBoxDirection": "normal",
                    "WebkitBoxOrient": "horizontal",
                    "WebkitBoxPack": "center",
                    "WebkitFlexDirection": "row",
                    "WebkitJustifyContent": "center",
                    "alignItems": "center",
                    "flexDirection": "row",
                    "justifyContent": "center",
                    "msFlexAlign": "center",
                    "msFlexDirection": "row",
                    "msFlexPack": "center",
                  }
                }
              >
                <div
                  className="css-text-901oao css-textOneLine-bfa6kz"
                  dir="auto"
                  style={
                    Object {
                      "color": "rgba(0,0,0,0.32)",
                      "direction": "ltr",
                      "fontFamily": "Roboto-Medium, Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                      "letterSpacing": "1px",
                      "marginBottom": "9px",
                      "marginLeft": "8px",
                      "marginRight": "8px",
                      "marginTop": "9px",
                      "textAlign": "center",
                    }
                  }
                >
                  <div
                    className="css-view-1dbjc4n"
                    style={
                      Object {
                        "WebkitBoxPack": "center",
                        "WebkitJustifyContent": "center",
                        "display": "inline-flex",
                        "justifyContent": "center",
                        "minHeight": "34px",
                        "msFlexPack": "center",
                      }
                    }
                  >
                    <span
                      className="css-text-901oao css-textHasAncestor-16my406"
                      dir="auto"
                      style={
                        Object {
                          "color": "rgba(255,255,255,1.00)",
                          "direction": "ltr",
                          "fontFamily": "Roboto",
                          "fontSize": "16px",
                          "fontWeight": "normal",
                          "lineHeight": "22px",
                          "textAlign": "center",
                          "textDecoration": "none",
                          "textTransform": "uppercase",
                        }
                      }
                    >
                      Skip
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
